bylaws.md

```diff
@@ -1,14 +1,17 @@
 <!-- multilingual suffix: en, da, se -->
-<!-- [common] -->
-This document has multiple purposes. First, it functions as  the bylaws for a local chapter as required by the ACM. Secondly, it functions as official documentation required to register the organisation in both the danish and swedish business register.
-
-The document is also an exercise in pratical multilingualism and cross-border collaboration.
+<!-- [en] -->
+# Bylaws
+This document contains the bylaws for a local chapter of Association for Computing Machinery, hereafter referred to as ACM. In addition this document contains the information required for the registration of voluntary organisations in the danish and swedish business registers.
 
 <!-- [en] -->
 # Name and location
-The name of the organisation is Oeresund Chapter of Association for Computing Machinery, sometimes reffered to as ACM, and is also using the local names ACM Øresund og ACM Öresund.
+The name of the organisation is Oeresund Association for Computing Machinery Chapter and is using the local names of ACM Øresund and ACM Öresund. Hereafter referred to as The Chapter.
+
+The ACM is located at 100 West Tenth Street in the City of Wilmington, County of New Castle, USA. The name and address of the resident agent is the Corporation Trust Company, 100 West Tenth Street, Wilmington, Delaware, USA.
 
-The ACM is located at 100 West Tenth Street in the City of Wilmington, County of New Castle. The name and address of the resident agent is the Corporation Trust Company, 100 West Tenth Street, Wilmington, Delaware.
+The Chapters mailing address is Vesterbrogade 999, 1620 København V, Denmark.
```

Comment:
Address required for voluntary organisation registration in Denmark but might not have to go in the bylaws.